webapp/components/Editor/nodes/Hashtag.js

```diff
@@ -25,7 +25,7 @@ export default class Hashtag extends TipTapMention {
           'a',
           {
             class: this.options.mentionClass,
-            href: `/search/hashtag/${node.attrs.id}`,
+            href: `/search/hashtag/${encodeURI(node.attrs.id)}`,
```

Comment:
Can't we use query parameters? :cry: 

I found out how to use [URL](https://nodejs.org/api/url.html) to encode/decode url params for relative URLs: Just use a dummy domain!

Example:

```js
const url = new URL('http://example.org')
url.searchParams.append('hashtag', 'ä  #')
url.search // returns ?hashtag=%C3%A4++%23
```